spec/models/warehouse/case_report_spec.rb

```diff
@@ -30,17 +30,19 @@
       expect(kase.warehouse_case_report).to be_nil
 
       case_report = described_class.for(kase)
-      expect(case_report.new_record?).to be true
+      kase.reload
+
+      expect(kase.warehouse_case_report).to eq case_report
     end
 
-    it 'returns existing CaseReport if it exists' do
+    fit 'returns existing CaseReport if it exists' do
```

Comment:
what is fit?

Reply:
balls, sorry - that's rspec shorthand for focusing a test. Really bad that I left it in there - it prevents other tests in the file from executing

Reply:
Removed!